cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -82,3 +87,64 @@ def is_ansible_pod_stopped(enabled_embedded_appliance):
         assert wait_for(is_nginx_stopped, func_args=[enabled_embedded_appliance], num_sec=30)
     else:
         assert wait_for(is_ansible_pod_stopped, func_args=[enabled_embedded_appliance], num_sec=180)
+
+
+@pytest.mark.tier(1)
+def test_embedded_ansible_event_catcher_process(appliance, enabled_embedded_appliance):
+    """
+    EventCatcher process is started after Ansible role is enabled (rails
+    evm:status)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/4h
+        tags: ansible_embed
+    """
+    result = appliance.ssh_client.run_rake_command(
+        "evm:status | grep 'EmbeddedAnsible'"
+    ).output
+
+    for data in result.splitlines():
+        logger.info("Checking service/process %s started or not", data)
+        assert "started" in data
+
+
+@pytest.mark.tier(1)
+def test_embedded_ansible_logs(appliance, enabled_embedded_appliance):
+    """
+    Separate log files should be generated for Ansible to aid debugging.
+    p1 (/var/log/tower)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/4h
+        tags: ansible_embed
+    """
+    log_checks = [
+        "callback_receiver.log",
+        "dispatcher.log",
+        "fact_receiver.log",
+        "management_playbooks.log",
+        "task_system.log",
+        "tower.log",
+        "tower_rbac_migrations.log",
+        "tower_system_tracking_migrations.log",
+    ]
+
+    # Asserting log folder is present
+    tower_log_folder = appliance.ssh_client.run_command("ls /var/log/tower/")
+    assert tower_log_folder.success
+
+    # Removing setup log file from list and asserting it separately
+    # Setup log file contains date/time string in it.
+    logs = tower_log_folder.output.splitlines()
+    setup_log = logs.pop(4)
```

Comment:
I am suspect here how you know index `4` pointing `setup.log` file only?

Reply:
@digitronik After converting `Unicode` into `List`, every time I am getting `setup*` file location at 4. 3-4 times I ran locally as well as on `miq shell` and confirmed that it is appearing at index 4 only.